### ISSUE TITLE
Ideas for creation challenges

### ISSUE THREAD

**Issue by LeonieHagitte** [brandmaier/shiny-workshop-2023#1]
Box-plots / or something similar
Regression/ trends over time or similar stuff with graphs

Images (e.g. like the interactive brain images on the website https://shiny.rstudio.com/gallery/mri-images.html)

**Comment by LeonieHagitte** [comment]
we should think about for what context we want to present challenges - like is this meant for a nice way of presenting research data, or is it meant for education, or generally how much interaction do we want to include- maybe we could use this to structure the challenges thereby?

**Comment by brandmaier** [comment]
interactive brain imaging is awesome but maybe too specific for the crowd?

**Comment by brandmaier** [comment]
What is the simplest app we can start with? A pocket calculator that adds numbers?

**Comment by brandmaier** [comment]
Here is a sketch for a simple calculator that people could try to extend... https://github.com/brandmaier/shiny-workshop-2023/commit/eda5373bf497e98b898d914e177da3cf6e821ee6

**Comment by LeonieHagitte** [comment]
yes, for a first practice I think that is nice, maybe they could extend this with more operators/ operations, like multiplication etc.?

**Comment by LeonieHagitte** [comment]
I just thought that in the end it would be cool, if everyone would have done something, they could use in their studies etc.?
